example/ios/Podfile

```diff
@@ -62,6 +62,9 @@ target 'Runner' do
   }
 end
 
+# Prevent Cocoapods from embedding a second Flutter framework and causing an error with the new Xcode build system.
```

Comment:
Is this a cocoa pods bug?

Reply:
its actually a bug in xcode's new build system afaik